bricks/vs/src/code-editor/index.tsx

```diff
@@ -208,6 +209,15 @@ class CodeEditor extends FormItemElementBase implements CodeEditorProps {
   })
   accessor tokenConfig: TokenConfig | undefined;
 
+  /**
+   * 在 brick_next_yaml 中是否开启语义相关校验
+   * @default false
+   */
+  @property({
+    type: Boolean,
+  })
+  accessor enableCustomValidationInBrickNextYaml: boolean | undefined;
```

Comment:
属性有点太长，`enable` 属于比较多余的

```suggestion
  accessor customValidationInBrickNextYaml: boolean | undefined;
```

Reply:
done